compiler-core/templates/docs-css/index.css

```diff
@@ -471,6 +472,10 @@ body.drawer-open .label-closed {
   margin-top: var(--large-gap);
 }
 
+.module-members:last-of-type .member:last-of-type {
```

Comment:
This seems like quite an unwieldy selector. Is there a single class or element we can reference? Perhaps adding padding to the bottom of the page?

Reply:
Yeah it's bad... at first I simply added the bottom margin to `.content` but that would result in a quite big bottom margin in the code section: `.member` already has some margin so this rule is removing the already existing margin for just the last bit of code of the page.

What would you do in this case? There's no single class to reference since I just want to remove the margin of the last member of the last module of a page

Reply:
`.content *:last-child` perhaps? It expresses the intention better too I think as it's relating to the margin added to `.content`.

Reply:
The problem is that `.content > *:last-child` would do nothing on the margin of the code page: it selects the last `.module-member` in the code page but it's not the responsible for the bottom margin (the element adding the margin is the `.member` _inside_ the `.module-member`). Otherwise the any page with code would still have a big chin:
![Screenshot 2023-06-23 alle 12 05 47](https://github.com/gleam-lang/gleam/assets/20598369/7119e361-a391-4a88-832d-6077c2d56be2)

I mean, it's not the worst thing we could also just accept it removing it is not worth the extra complexity in the `index.css`'s rules

Reply:
Let's go with it as it is then :)